soc/arm/nordic_nrf/Kconfig

```diff
@@ -16,6 +16,12 @@ config SOC_FAMILY
 source "soc/arm/nordic_nrf/Kconfig.peripherals"
 source "soc/arm/nordic_nrf/*/Kconfig.soc"
 
+config SOC_SECURE_NRF
```

Comment:
Shouldn't this be a global instead? in the arch/arm folder? Why is this Nordic-specific?

Reply:
So far soc_secure is a nordic only header. And it uses the TF-M platform IOCTL service, which contain platform specific services.